### PR TITLE
WIP: DEV-7805: first cut of order creation tool

### PR DESCRIPTION
**Pull request by timbarrass**
# Pull Request Checklist

- [ ] Read the [contributing guidelines](https://github.com/finbourne/lusid-python-tools/blob/master/docs/CONTRIBUTING.md)
- [ ] Tests pass

# Description of the PR

New lusidtools tool to create orders. Reads an order per row from a csv file.

### CODE REVIEW

File: lusidtools/lpt/create_orders.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                ),
+                inplace=True,
+            )
+
+        prop_keys = [col for col in df.columns.values if col.startswith("P:")]
+
+        identifiers = [col for col in df.columns.values if col in args.identifiers]
+
+        # Identifiers have to be unique
+        df = df.drop_duplicates(identifiers)
```
Comment (MikeMcGarry) [inline lusidtools/lpt/create_orders.py:64]: These lines are an exact copy of lines 18-67 in create_instr.py. Is it possible to move both of these functions into a common module that can be imported by both create_instr.py and create_orders.py?

File: lusidtools/lpt/create_orders.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+from lusidtools.lpt import stdargs
+
+NAME = "name"
+TOOLNAME = "orders_create"
+TOOLTIP = "Create Orders"
+
+
+def parse(extend=None, args=None):
+    return (
+        stdargs.Parser("Upsert Orders", ["filename", "limit", "test"])
```
Comment (MikeMcGarry) [inline lusidtools/lpt/create_orders.py:14]: Should this match the Tooltip? i.e. "Create Orders" in both or "Upsert Orders" in both.

File: lusidtools/lpt/create_orders.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        order_request=api.models.OrderSetRequest(
+            order_requests=[
+                api.models.OrderRequest(
+                    id=api.models.ResourceId(row["id.scope"], row["id.code"]),
+                    side=row["side"],
+                    quantity=row["quantity"],
+                    order_book_id=api.models.ResourceId(row["orderBookId.scope"], row["orderBookId.code"]),
+                    portfolio_id=api.models.ResourceId(row["portfolioId.scope"], row["portfolioId.code"]),
+                    instrument_identifiers={'Instrument/default/' + identifier: row[identifier] for identifier in identifiers},
+                    properties={key[2:]: api.models.ModelProperty(key[2:], api.models.PropertyValue(row[key])) for key in prop_keys if pd.notna(row[key])}
```
Comment (MikeMcGarry) [inline lusidtools/lpt/create_orders.py:79]: We have two different implementations for loading data from a file. One in the lpt folder which is designed to be called from the command line and one in the cocoon folder which is designed to be called using code. For this addition, what is the use case? Is there scope to create it using the cocoon implementation (which has more flexibility on names in the file and takes a configuration to map from the file to LUSID) or is there a pressing need to get this live with this hard coded implementation?

Comment (MikeMcGarry) [reply lusidtools/lpt/create_orders.py:79]: As discussed will go with the lpt implementation for now.

File: lusidtools/lpt/create_orders.py (hunk 1 above)
Comment (timbarrass) [reply lusidtools/lpt/create_orders.py]: Have factored out most into a new method in lpt.py

File: lusidtools/lpt/create_orders.py (hunk 2 above)
Comment (timbarrass) [reply lusidtools/lpt/create_orders.py]: Done

**Pull request merged**